src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -1573,6 +1584,17 @@ public void updateEnabledState() {
             getBackAction().setEnabled(false);
             getForwardAction().setEnabled(false);
         }
+
+        boolean saved = false;
+
+        if (tabCount > 0) {
```

Comment:
I just discovered that there is already a getCurrentBasePanel which does the same as the newly created `getSelectedPanel`.